keps/sig-cli/2906-kustomize-function-catalog/kep.yaml

```diff
@@ -24,10 +24,10 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.24"
+latest-milestone: "v1.27"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
-  alpha: "v1.24"
-  beta: "v1.25"
-  stable: "v1.27"
+  alpha: "v1.28"
+  beta: "v1.29"
+  stable: "v1.30"
```

Comment:
nit: these are super provisional and don't mean much until we can target kubectl... but fwiw we need at least two releases (of kubectl specifically) between beta and GA.